asm/macros/event.inc

```diff
@@ -2338,3 +2338,21 @@
 	.macro hideitemdescription
 	callnative ScriptHideItemDescription
 	.endm
+
+   .macro increasedifficulty
+    callnative Script_IncreaseDifficulty
+   .endm
+
+    .macro decreasedifficulty
+    callnative Script_DecreaseDifficulty
+    .endm
```

Comment:
we use TAB instead of spaces for this file

Reply:
[`b4dffb6` (#5337)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5337/commits/b4dffb64dc00aae93e03b139b88530cd8194c220)
[`88b0894` (#5337)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5337/commits/88b0894c6b436bb7604911ed704feae0749cfb0b)